mods/example/mod.yaml

```diff
@@ -6,16 +6,24 @@ Packages:
 	.
 	$example: example
 	./mods/common: common
-	example|bits
 
 MapFolders:
 	example|maps: System
 
 Rules:
-	example|rules.yaml
+	example|rules/dummyactor.yaml
+	example|rules/mpspawn.yaml
```

Comment:
Putting actors into individual rule files is a bit non-standard, but it's only 2 so it shouldn't matter.

Reply:
This is a good standard IMO that more mods should follow.